doc/man/salt.7

```diff
@@ -9889,6 +9889,21 @@ syndic_finger: \(aqab:30:65:2a:d6:9e:20:4f:d8:b2:f3:a7:d4:65:50:10\(aq
 .fi
 .UNINDENT
 .UNINDENT
+.SS \fBno_proxy\fP
```

Comment:
The man pages are automatically generated so they don't need to be included in PRs. It does no harm if they are, but just for future reference. :]